src/Platforms/SqlitePlatform.php

```diff
@@ -328,14 +328,34 @@ protected function _getCommonIntegerTypeDeclarationSQL(array $column)
         return ! empty($column['unsigned']) ? ' UNSIGNED' : '';
     }
 
+    /**
+     * Enables or disables schema emulation (replace dot with 2 underscores).
+     *
+     * Schema emulation is enabled by default to maintain backwards compatibility.
+     * Disable it to opt-in to the behavior of DBAL 4.
+     *
+     * @deprecated Will be removed in DBAL 4.0.
+     */
+    public function setSchemaEmulationEnabled(bool $schemaEmulationEnabled): void
```

Comment:
Why should the end-user care about setting some flag? A more natural method name might look like `enableSchemaEmulation()` and `disableSchemaEmulation()`.

Reply:
As this method is to be removed later or sooner and already marked as deprecated, I belive this is fine. If some change is still wanted, plese let me know.

Reply:
Yes, the requested change is still wanted.